ParisBNF/abb/BNFabb195.xml

```diff
@@ -0,0 +1,683 @@
+            <locus from="9r" to="16v"/>
+            </item>
+              <item xml:id="q4" n="3">
+            <dim unit="leaf">6</dim>
+          <locus from="17r" to="22v"/>
+                </item>
+
+        <item xml:id="q5" n="4">
+            <dim unit="leaf">8</dim>
+            <locus from="23r" to="30v"/>
+             </item>
+
+        <item xml:id="q6" n="5">
+          <dim unit="leaf">8</dim>
+        <locus from="31r" to="38v"/>
+        </item>
+      <item xml:id="q7" n="6">
+      <dim unit="leaf">8</dim>
+        <locus from="39r" to="46v"/>
+    </item>
+    <item xml:id="q8" n="7">
+        <dim unit="leaf">8</dim>
+      <locus from="47r" to="54v"/>
+     </item>
+     <item xml:id="q9" n="8">
+       <num value="5">፭</num>
+      <dim unit="leaf">8</dim>
+      <locus from="55r" to="62v"/>
+     </item>
+<item xml:id="q10" n="9">
+  <dim unit="leaf">8</dim>
+  <locus from="63r" to="70v"/>
+   </item>
+
+  <item xml:id="q11" n="10">
+    <dim unit="leaf">8</dim>
+    <locus from="71r" to="78v"/>
+     </item>
+
+<item xml:id="q12" n="11">
+  <dim unit="leaf">7</dim>
+  <locus from="79r" to="85v"/>
+  <note>The structure of the quire cannot
+  be discerned.</note>
+  </item>
+
+  <item xml:id="q13" n="12">
+   <dim unit="leaf">8</dim>
+     <locus from="86r" to="93v"/>
+  </item>
+  <item xml:id="q14" n="13">
+   <dim unit="leaf">8</dim>
+     <locus from="94r" to="101v"/>
+  </item>
+
+ <item xml:id="q15" n="14">
+<dim unit="leaf">6</dim>
+  <locus from="102r" to="107v"/>
+  </item>
+  <item xml:id="q16" n="15">
+    <dim unit="leaf">6</dim>
+  <locus from="108r" to="113v"/>
+     </item>
+
+    <item xml:id="q17" n="16">
+        <dim unit="leaf">10</dim>
+      <locus from="114r" to="123v"/>
+      </item>
+
+   <item xml:id="q18" n="17">
+      <dim unit="leaf">4</dim>
+      <locus from="124r" to="127v"/>
+    </item>
+  </list>
+  </collation>
+      <condition key="good">
+        <note>The text block has been trimmed, probably in Europe.</note>
+      </condition>
+  </supportDesc>
+      <layoutDesc>
+        <layout columns="3" writtenLines="27" xml:id="textarea1">
+      <locus from="1r" to="126v"/>
+      <note corresp="#leafdim">Data taken from <locus target="#17r"/>.</note>
+        </layout>
+
+
+        <layout columns="3" writtenLines="28" xml:id="textarea2">
+            <locus target="#116v"/>
+          <note corresp="#leafdim">Data taken from fol. 116v.</note>
+            </layout>
+            </layoutDesc>
+              </objectDesc>
+
+                <handDesc>
+              <handNote script="Ethiopic" xml:id="h1" corresp="#ms_i1 #ms_i2
+            #ms_i3 #ms_i4 #ms_i5 #ms_i6 #ms_i7 #ms_i8 #a2">
+                  <date notBefore="1800" notAfter="1840"/>
+                  <seg type="ink">Black, red (light vermillion)</seg>
+                  <desc>Written in a skilled but irregular,
+                    hasty hand. Small script, ca. 20-22 signs per line.
```

Comment:
out of curiosity, why signs and not characters?

Reply:
No special reason, but I can change if you think "character" is prevailing in the DB and should be preferably used for consistency.

Reply:
It's just what we used for DS. Later in this record, there is "letters". I was wondering whether there might be a conceptual difference (counting or not word separator or so).

Reply:
There might be; I used to use one general term for all characters applied in the entire Ethiopic writing system (including word dividers and etc., when we count characters per line, we count everything), while letters refer only to letters, numerals to numerals etc.